book/src/SDK/platform/mongodb/migrations.md

```diff
@@ -0,0 +1,55 @@
+## Migrations
+
+Sometimes, it is necessary to refactor a data model after it has already been release to 
+production. The `migrations` module provides a mechanism to both `apply` and `rollback` a migration.
+This mechanism is limited in its opinions and feature set, since migrations are highly 
+contextual by their nature. Migration script authors are responsible for ensuring that their 
+migrations are either idempotent and safe to re-run, or for checking the migration log to 
+determine if a script has already been applied.
+
+### Migration Component Model
```

Comment:
I'm pretty sure nobody has any experience with the code here for migrations.  At some point, it would be great to get more acquainted with this area of the system.